packages/compiler-sfc/src/rewriteDefault.ts

```diff
@@ -13,33 +10,23 @@ const exportDefaultClassRE =
 export function rewriteDefault(
   input: string,
   as: string,
-  parserPlugins?: ParserPlugin[]
+  /** @deprecated use `ast` param instead */
+  parserPlugins?: ParserPlugin[],
```

Comment:
`rewriteDefault` is an exported and open API. So for avoiding a breaking change, I kept the `parserPlugins` and mark it `deprecated`.